udfs/community/test_cases.yaml

```
@@ -408,6 +408,11 @@ levenshtein:
   - test:
     input: CAST('Connecticut' AS STRING), CAST('CT' AS STRING)
     expected_output: CAST(10 AS INT64)
+pi:
+  - test:
+    input: null
```

NULL is a type in YAML so wrap it in parenthesis
```suggestion
    input: (NULL)
```

---

Thank you @danieldeleo ! This helps alot!